### ISSUE TITLE
[Bug] Changes to temporalcloud_namespace with custom search attributes failing

### ISSUE THREAD

**Issue by KoehlerD-SSG** [temporalio/terraform-provider-temporalcloud#75]
### What are you really trying to do?

Update a property on a namespace that has a custom search attribute

### Describe the bug

The in-place update operation for the `temporalcloud_namespace`  resource seems to attempt to delete the namespace's search attribute, meaning namespaces with custom search attributes can't be updated (as custom search attributes can't be deleted). 

(I haven't yet had a chance to look at the provider code / Ops API it's using)

### Minimal Reproduction

```hcl
resource "temporalcloud_namespace" "test" {
  name               = "tf-test-repro"
  regions            = ["aws-ca-central-1"]
  retention_days     = 90
  accepted_client_ca = base64encode(file("${path.module}/SomeCA.cert"))
}

resource "temporalcloud_namespace_search_attribute" "some_attribute_name" {

  namespace_id = temporalcloud_namespace.test.id
  name         = "some_attribute_name"
  type         = "Keyword"
}
```

```diff
-  retention_days     = 90
+  retention_days     = 89
```
Will plan something like:
```
  # temporalcloud_namespace.test will be updated in-place
  ~ resource "temporalcloud_namespace" "test" {
        id                 = "tf-test-repro.xyzzz"
        name               = "tf-test-repro"
      ~ retention_days     = 90 -> 89
        # (2 unchanged attributes hidden)
    }
```
```
temporalcloud_namespace.test: Modifying... [id=tf-test-repo.xyzzz]
╷
│ Error: Failed to update namespace
│ 
│   with temporalcloud_namespace.test,
│   on main.tf line 25, in resource "temporalcloud_namespace" "test":
│   25: resource "temporalcloud_namespace" "test" {
│ 
│ rpc error: code = InvalidArgument desc = deleting an existing attribute 'some_attribute_name' is not supported
```

**Comment by swgillespie** [comment]
that's not good - thank you for the report! We will check this out.

**Comment by swgillespie** [comment]
Thank you - we will publish a release with the fix today!

**Comment by swgillespie** [comment]
@KoehlerD-SSG the fix for this went out in version `v0.0.5`
